doc/api/fs.md

```diff
@@ -478,6 +478,56 @@ If one or more `filehandle.read()` calls are made on a file handle and then a
 position till the end of the file. It doesn't always read from the beginning
 of the file.
 
+#### `filehandle.readLines([options])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `options` {Object}
+  * `encoding` {string} **Default:** `null`
+  * `autoClose` {boolean} **Default:** `true`
+  * `emitClose` {boolean} **Default:** `true`
+  * `start` {integer}
+  * `end` {integer} **Default:** `Infinity`
+  * `highWaterMark` {integer} **Default:** `64 * 1024`
+* Returns: [`readlinePromises.Interface`][]
+
+Convenient alias to create a `readline` interface and stream over the file. See
+[`filehandle.createReadStream()`][] for the options.
+
+```mjs
+import { open } from 'node:fs/promises';
+
+let file;
+try {
+  file = await open('./some/file/to/read');
+
+  for await (const line of file.readLines()) {
+    console.log(line);
+  }
+} finally {
+  await file?.close();
```

Comment:
Could this be handled automatically by the async iterator?

Reply:
Oh it is indeed, `autoClose` defaults to `true` so that's indeed unnecessary.